mujoco_py/tests/test_modder.py

```diff
@@ -49,28 +49,28 @@ def test_textures():
     modder.set_rgb('g2', (0, 255, 0))
     modder.set_rgb('g3', (0, 0, 255))
     modder.set_rgb('g4', (255, 0, 255))
-    compare_imgs(sim.render(200, 200, camera_name="topcam"),
+    compare_imgs(sim.render(201, 205, camera_name="topcam"),
```

Comment:
I would add that i think its _slightly_ better coverage to have width != height and to have odd numbers.